index.html

```diff
@@ -5,8 +5,198 @@
+                <li><a href="#">Events</a></li>
+                <li><a href="#">Documentation</a></li>
+                <li><a href="about.html">About</a></li>
+            </ul>
     
+        </div>
+
+    </nav>
+    <main class="main">
+        <section class="headline">
+            <div class="headline-opacity">
+                <header class="headline-header">
+                    <h2>"Hello Sharing world"</h2>
+                    <h1>Africa's Talking Open Hackathon: FinTech</h1>
+                </header>
+                <div class="headline-desc">
+                    <p>
+                        Join us at our open hackathon every last Thursday of the month.
+                         Come build with us, present feature request, give feedback or 
+                         just contribute to our open source API's and SDKs. 
+                         Get your ticket, let us exchange knowledge, 
+                         best practices, submit feature requests and innovate.    
+                    </p>
+                </div>
+                <div class="headline-date">
+                    <h2>THU,Jun 30, 9:00 AM (EAT)</h2>
+                    <h5>@ Africa's Talking HQ
+                        Mzima Springs Rd, Lavington, Nairobi
+                        Nairobi, 00100</h5>
+                </div>
+            </div>
+        </section>
+        <section class="main-program-section">
+            <header class="program-header">
+                <h3>Agenda</h3>
+                <div class="divider"></div>
+            </header>
+            
+            <div class="articles">
+                <article class="program-article">
+
+                    <div class="program-img">
+                        <img src="https://img.icons8.com/ios/50/FFFFFF/functional-training.png"/>                    </div>
+                    <div class="p-name">
+                        <h3>Warm-up</h3>
+                    </div>
+                    <div class="p-desc">
+                        <p>
+                            Introduction and Pitching, 
+                            Form teams where need be. 
+                            Building/hacking, 
+                            Feedback, 
+                            Feature requests.
+                        </p>
+                    </div>
+                </article>
+                <article class="program-article">
+                    <div class="program-img">
+                        <img src="https://img.icons8.com/ios/50/FFFFFF/laptop-coding.png"/>                    </div>
+                    <div class="p-name">
+                        <h3> Code</h3>
+                    </div>
+                    <div class="p-desc">
+                        <p>
+                            Code in GitHub/GitLab/BitBucket and deploy in Docker/Automated
+                             CI/Travis CI/Bamboo <br>
+                        </p>
+    
+                    </div>
+                </article>
```

Comment:
- [ ] **Required:** You have been doing a really hard job, don't give up now. I want to encourage you to keep going. While I would love to list out all you should do I observed that @Ismailco had done a really good job doing that.

Going forward in the program, I encourage you to seek to know the goal of every milestone. While most of the time you'd be encouraged to do personalized and completely creative design, this milestone's goal is to see how you can stick to a given professional design and produce a look-alike amongst other things.

Having said the above, I suggest you relax and go over the design step by step again and make them look like the given template. Added to the comments of the previous reviewers, I'll give you some observations to begin with below:

Your Design | Template
:---: | :---:
![image](https://user-images.githubusercontent.com/27709832/175756880-21e13d2d-eb25-4c38-aea8-0a34ceef1f4a.png) | ![image](https://user-images.githubusercontent.com/27709832/175756906-37f05cb3-6e12-4a4d-a667-7878b625f8d9.png)

**_Nubering is according to the label on the image above_**

1. The `font-weight` is lighter in the CC template than in your design and you have used a different `font-family` 
2. The design you have currently is styled completely differently from the given design.
3. Your `font-family` for the small text "Hello sharing world" and the **h1** text should be as given. Additionally, the h1 has a background image. More so, the section has a background image.
4. The `font-family` is also different.

**_The above list is just to give you a starting point_**. Go through every element and style them accordingly.

I really hope this helps. 😄